CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.0] - 2022-XX-XX
```

Comment:
https://github.com/igiagkiozis/plotly/blob/c0ca69192c422862941617a72094e4e44fc9b67f/CONTRIBUTING.md?plain=1#L34-L35 asks me to do this. Yet, for example #99 was merged without these changes. Please let me know if I should remove / amend these changes.